deployments/k8s/clusterrole.yaml

```diff
@@ -29,6 +29,14 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
Config map permissions was split out to a separate role: https://github.com/signalfx/signalfx-agent/blob/master/deployments/k8s/configmap-role.yaml.  

Perhaps you were using an older Helm chart?  The latest has been released as of a few days ago.  

This should be unnecessary.  The agent should not access configmaps outside of its own namespace, which is why we split out that permission to a separate role and not a cluster role.